src/base/utils.js

```diff
@@ -235,10 +235,34 @@ export function removeArrayItem(arr, item) {
   }
 }
 
+// HTML5 video element recycle storage object.
+// If playback recycleVideo flag is set, it store
+// one video element for each player instance created.
+const videoStore = {}
+
+export class VideoStore {
```

Comment:
@towerz what are the requirements for have more than one video element in single player ? _(i have not yet find any infos to reproduce this behaviour)_